### ISSUE TITLE
[BUG] Critical Vehicle flying after going out of an apartment 

### ISSUE THREAD

**Issue by milotiro95** [qbcore-framework/qb-apartments#79]
I have some player going outside the classic appartments and when they enter the car it goes crazy and fly until it goes off the car. Only fix is not to enter classic apartments and restart fivem ... Impossible to fix the issue for now and don't understand how it's possible also.

**Comment by milotiro95** [comment]
![Sans titre](https://user-images.githubusercontent.com/86166328/169617824-6049e162-5fa7-44fd-957d-a68e1c8b68a4.png)

**Comment by milotiro95** [comment]
Impossible to stop except /logout or restart fivem

**Comment by milotiro95** [comment]
This kind of bug https://streamable.com/0egstg (not on my server but exactly the same problem)

**Comment by c0deMach1ne** [comment]
**Could it be because of the script that allows you to carry the guns on your back?**

**Comment by milotiro95** [comment]
Mmh It can be but I never had issue before with weaponback and in fact I have weaponback on my zombie server and not on my rp server and the issue is happening in the two servers

**Comment by omar-othmann** [comment]
this problem is when you incoming a call and you answer it, then when you enter the apartment and the phone in your hand(in call)

so the problem is from phone animation and enter apartment animation. is playing together,

to fix it, you should comment on enter apartment animation or wait for resource owner to fix it.

Thanks.

**Comment by milotiro95** [comment]
I just try the way you did and exactly the problem happens. Damn finally so much time to identify this bug ... It looks like it broke the phone in the hands and it stay attached to your hands

**Comment by omar-othmann** [comment]
just comment on it like this

```local function openHouseAnim()
    -- loadAnimDict("anim@heists@keycard@")
    -- TaskPlayAnim( PlayerPedId(), "anim@heists@keycard@", "exit", 5.0, 1.0, -1, 16, 0, 0, 0, 0 )
    -- Wait(400)
    -- ClearPedTasks(PlayerPedId())
end```

**Comment by milotiro95** [comment]
In qb-appartments, qb-houses and qb-houserobbery is that right ?

**Comment by milotiro95** [comment]
I have try and this fix not working :/

**Comment by milotiro95** [comment]
For now I have disable all animations in qb-phone waiting for a fix ..

**Comment by Maverick167** [comment]
> I just try the way you did and exactly the problem happens. Damn finally so much time to identify this bug ... It looks like it broke the phone in the hands and it stay attached to your hands. Do you have an idea how to fix this ? Perhaps remove the animations parts ?

I can confirm that this DOES happen to people even when they aren't messing with their phones. Not sure why "bug" was changed to "invalid". It's a real problem that is quite annoying and it absolutely has to do with people entering their apartments and accessing stashes.

**Comment by milotiro95** [comment]
Disable all phones animations fix the problem tested it for 2 weeks without issues. Waiting for a fix from qb-phone authors

**Comment by Howsn** [comment]
Anny news on this?

**Comment by nzkfc** [comment]
We don't have this issue, never have when leaving apartment. The reason for this is due to prop colliision on the phone and then getting into a car, using /refreshskin instantly fixes this issues if you are flipping out.

However i've not been able to re-create this in months using qb-phone.

**Comment by nzkfc** [comment]
Yeah was phone prop from 3rd party phone script sometimes, is there a way
to stop props being stuck when entering cars that could be built in?

**Comment by RKDimitrov** [comment]
Any full fixes found? In my case it happens every time I open a stash in apartment. I dont have the weapons on the back script
